servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder/src/main/resources/application-mi.yaml

```diff
@@ -2,9 +2,9 @@ spring:
   cloud:
     azure:
       credential:
-        managed-identity-client-id: [ managed-identity-client-id ]
+        managed-identity-client-id: ${AZURE_MANAGED_IDENTITY_CLIENT_ID]
```

Comment:
change ${AZURE_MANAGED_IDENTITY_CLIENT_ID] to ${AZURE_MANAGED_IDENTITY_CLIENT_ID}, same for other places